pkg/tests/handlers/accounts.go

```diff
@@ -55,6 +56,34 @@ func TestRegisterNewCluster(options *helpers.TestOptions) error {
 	return nil
 }
 
+func TestRegisterExistingCluster(options *helpers.TestOptions) error {
+
+	testName := options.TestName
+	quantity := options.Rate.Freq
+	targeter := generateClusterReRegistrationTargeter(quantity, options.Path, options.Connection)
+
+	fileName := fmt.Sprintf("%s_%s.json", options.ID, testName)
+	resultFile, err := helpers.CreateFile(fileName, options.OutputDirectory)
+	if err != nil {
+		return err
+	}
+	defer resultFile.Close()
+
+	// Store Metrics from load test
+	options.Metrics[testName] = new(vegeta.Metrics)
+	defer options.Metrics[testName].Close()
+
+	for res := range options.Attacker.Attack(targeter, options.Rate, options.Duration, testName) {
+		result.Write(res, resultFile)
+		options.Metrics[testName].Add(res)
+	}
+
+	log.Printf("Results written to: %s/%s\n", options.OutputDirectory, fileName)
+
```

Comment:
Could you add the Write Report section
https://github.com/cloud-bulldozer/ocm-api-load/blob/main/pkg/tests/handlers/accounts.go#L48

Reply:
Sure!

Reply:
@chentex  I've added in the WriteReport section. Let me know if I am overlooking anything